tools/block_cache_pysim.py

```diff
@@ -0,0 +1,857 @@
+#!/usr/bin/env python3
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+
+import gc
+import random
+import sys
+import time
+from os import path
+
+import numpy as np
+
+
+kSampleSize = 16  # The sample size used when performing eviction.
+kMicrosInSecond = 1000000
+kSecondsInMinute = 60
+kSecondsInHour = 3600
+
+
+class TraceRecord:
+    """A trace record represents a block access."""
+
+    def __init__(
```

Comment:
Add a comment or linter rule. It's better to enforce parity between this and its C++ counterparts. If it's too hard, then I am fine with a comment linking these two, reminding to modify the other if one is being modified.

Reply:
As discussed offline, I added a comment inline.